src/rest-server/src/utils/error.d.ts

```diff
@@ -31,6 +31,7 @@ declare type Code =
     'ConflictUserError' |
     'ConflictVcError' |
     'ForbiddenUserError' |
+    'ForbiddenKeyError' |
```

Comment:
the wrong name in configuration seems like a config error instead of key error

Reply:
Use this name because user doesn't have permission to modify system key.